android/src/main/java/com/applovin/reactnative/AppLovinMAXNativeAdView.java

```diff
@@ -400,19 +400,11 @@ private void sendAdLoadedReactNativeEventForAd(final MaxNativeAd ad)
         if ( !Float.isNaN( aspectRatio ) )
         {
             // The aspect ratio can be 0.0f when it is not provided by the network.
-            if ( Math.signum( aspectRatio ) == 0 )
-            {
-                nativeAdInfo.putDouble( "mediaContentAspectRatio", 1.0 );
-            }
-            else
+            if ( Math.signum( aspectRatio ) != 0 )
```

Comment:
Is it expected to handle negative signs as well? Instead of `> 0`?

Reply:
Updated to check the negative signs.